common/firmware/adc/adc_comms.cpp

```diff
@@ -0,0 +1,18 @@
+#include "common/firmware/adc_comms.hpp"
```

Comment:
The same thing with the adc c file goes for this. This C++ file is entirely bound up in the concerns of the `head`, and should be in `head/` not `common/`